onnx/onnx.in.proto

```diff
@@ -99,6 +99,11 @@ message AttributeProto {
 
   // The name field MUST be present for this version of the IR.
   optional string name = 1;           // namespace Attribute
+ 
+  // if ref_attr_name is not empty, ref_attr_name is the attribute name in parent function.
+  // In this case, this AttributeProto does not contain data, and it's a reference of attribute
+  // in parent scope.
+  optional string ref_attr_name = 21;
```

Comment:
Need to mention that this is not currently valid in ModelProto

Reply:
Sure.